### PR TITLE
Add Bulgarian and update translation files

### PR DESCRIPTION
**Pull request by camilavcoutinho**
Add bulgarian, update translations and remove unwanted files

### CODE REVIEW

File: messages/ar.json [hunk 1]
```diff
@@ -1,60 +0,0 @@
-{
```
Comment (athoscouto) [inline messages/ar.json:1]: Was this removal intentional? Haven't seen it mentioned anywhere.

**Review by athoscouto — APPROVED** [review]
LGTM

File: messages/ar.json (hunk 1 above)
Comment (daniserejo) [reply messages/ar.json:1]: yes :D
